### ISSUE TITLE
Adding a "figure mode" button

### ISSUE THREAD

**Issue by fedarko** [biocore/emperor#753]
This would be a single button accessible in the UI that does the following things:

- changes the plot to 2D
- changes the background to white
- changes sample shapes to 2D circles (#607)
- ideally, would move axis labels off of the actual axes (this might be worth a separate issue)

Eventually I'd like to take a shot at this, but don't really have time right now... I think this would make a lot of people happy.

(Came up during discussion with some folks in the lab.)

**Comment by ElDeveloper** [comment]
This sounds like a great idea! This reminds me of the concept of "themes" which would be very cool to support.

**Comment by fedarko** [comment]
Agreed that the "themes" idea is super cool -- once we'd have this functionality implemented, I imagine that passing in arbitrary settings for most of the options (e.g. making the plot have a 3D red background with all squares... that sounds awful, but you get the idea :) wouldn't be a ton of extra work.

That could also be useful as an easy way to add a "switch back to the starting configuration" button.
